File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/13.1 - Nested Loops More Exersises/11. HappyCat Parking.py
count_days = int(input())
count_hours = int(input())

total = 0
daily_total = 0

for day in range(1, count_days + 1):
    for hour in range(1, count_hours + 1):
        if day % 2 == 0 and hour % 2 != 0:
            daily_total += 2.5
            total += 2.5
        elif day % 2 != 0 and hour % 2 == 0:
            daily_total += 1.25
            total += 1.25
        else:
            daily_total += 1
            total += 1
    print(f"Day: {day} - {daily_total:.2f} leva")
    daily_total = 0

print(f"Total: {total:.2f} leva")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/16. УПРАЖНЕНИЕ ВЛОЖЕНИ ЦИКЛИ/Nested Loops More Exercises/11. HappyCat Parking.js
function happyCatParking(input) {
    // Mask
    let days = Number(input.shift());
    let hours = Number(input.shift());

    let total = 0;

    for (let day = 1; day <= days; day += 1) {
        let price = 0;
        for (let hour = 1; hour <= hours; hour += 1) {
            if ((day % 2 == 0) && hour % 2 != 0) {price += 2.5;}
            else if ((day % 2 != 0) && (hour % 2 == 0)) {price += 1.25;}
            else {price += 1;}
        }
        total += price;
        console.log(`Day: ${day} - ${price.toFixed(2)} leva`);
    }

    console.log(`Total: ${total.toFixed(2)} leva`);
}

happyCatParking([2, 5]);  // Should return:
// Day: 1 - 5.50 leva
// Day: 2 - 9.50 leva
// Total: 15.00 leva

happyCatParking([5, 2]);  // Should return:
// Day: 1 - 2.25 leva
// Day: 2 - 3.50 leva
// Day: 3 - 2.25 leva
// Day: 4 - 3.50 leva
// Day: 5 - 2.25 leva
// Total: 13.75 leva
